apps/web/src/components/Router.tsx

```diff
@@ -185,7 +185,14 @@ const createRoute = (locale: Locales, featureFlags: FeatureFlags) =>
             },
             {
               path: "logged-out",
-              loader: () => {
+              loader: ({ request }) => {
+                const url = new URL(request.url);
+                const from = url.searchParams.get("from");
+                if (from) {
+                  // eslint-disable-next-line @typescript-eslint/only-throw-error
+                  throw redirect(from);
+                }
```

Comment:
On line 202 we do a `window.location.href` set.  Is there a reason why you did a `redirect` here?

Reply:
`redirect` sets the status code to `302` in the browser history which seemed like what we expect here. Though, if we do not want that behaviour, I can change it to the other method.